modules/rosa-sts-account-wide-role-and-policy-commands.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-about-iam-resources.adoc
+
+[id="rosa-sts-account-wide-role-and-policy-aws-commands_{context}"]
+= Account-wide IAM role and policy `aws` command reference
+
+This section references the `aws` CLI commands that are shown in the terminal when you run the following `rosa` command using `manual` mode:
+
+[source,terminal]
+----
+$ rosa create account-roles --mode manual --version 4.8
+----
+
+[NOTE]
+====
+When using `manual` mode, the `aws` commands are printed to the terminal for your review. After reviewing the `aws` commands, you must run them manually. Alternatively, you can specify `--mode auto` with the `rosa create` command to run the `aws` commands immediately.
+====
+
+.
+[source,terminal]
+----
+aws iam create-role \
```

Comment:
Yes, this is the output of the preceding command. I originally had `.Example output` as a title. But, in a conversation with @0kashi we decided to use something other than the word "Example", because the commands need to be used exactly as they are if they are subsequently run manually.

I will add another title here and in the other relevant modules.